### ISSUE TITLE
Need option to make all eyes different colours

### ISSUE THREAD

**Issue by hardikbhalodi** [alexzhirkevich/custom-qr-generator#29]
Is it possible to give different colours to eyes? right now colour applying to all eyes.

**Comment by alexzhirkevich** [comment]
Not possible at the moment and is not likely to be. Too colourful QR codes are hardly recognisable for scanners. Avoid doing it

**Comment by alexzhirkevich** [comment]
Will be possible after #35 with this code:

```kotlin
class DifferentColor() : QrVectorColor {

    private var number = 0
    
    override val mode: QrPaintMode = QrPaintMode.Separate

    override fun Paint.paint(width: Float, height: Float) {
        color = when (number % 3 /*4*/) {
            0 -> Color.RED
            1 -> Color.BLUE
            2 -> Color.GREEN
            //4 -> Color.BLACK  if fourth eye is enabled
            else -> 0
        }
        number++
    }
}
```

**Comment by alexzhirkevich** [comment]
Available in `2.0.0-alpha01` with the code above
